Support for Maven Assembly to build archives

rbarbey opened the pull request:
For our Maven-based build, we use the Maven assembly plug-in to produce a ZIP archive which is later inflated and copied to right directory when the build the main distribution package. Until now, we added a `pom.xml` and an assembly descriptor to each our Ember projects, building the Maven artifacts remained cumbersome. To get direct support via`ember deploy` I added support for producing such archives to this add-on.

Please review this pull request and let me know what I can do to improve it. I tried to follow your coding conventions. Unfortunately I missed your request to follow the conventional commits. Sorry about that.

makepanic commented:
Hey thanks for the PR, 
I probably can't review it before the weekend but will do it as soon as possible.

rbarbey commented:
No worries, take your time!

> Am 12.09.2018 um 13:36 schrieb Christian <notifications@github.com>:
> 
> Hey thanks for the PR,
> I probably can't review it before the weekend but will do it as soon as possible.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

rbarbey commented:
Thanks for the review. I addressed the two issues, moved the template creation to the top and replaced the links to our internal registry (should have matched that myself, sorry).

makepanic commented:
Thanks for the hard work. I've merged and released it as `0.2.0`
